pynest/nest/raster_plot.py

```diff
@@ -360,3 +360,14 @@ def show():
     See also: http://matplotlib.sourceforge.net/faq/howto_faq.html#use-show
     """
     pylab.show()
+
+def savefig(fname=str(uuid.uuid4()), **kwargs):
```

Comment:
I think that the random default filename is not a good idea for two reasons:
1. this might leave heaps of unclear files lying around in looped situation, which no one will recall being the raster plots after three months
2. according to [the pylab.showfig() documentation](https://matplotlib.org/devdocs/api/_as_gen/matplotlib.pyplot.savefig.html), the file format is deduced from the file name if no format is given. This does not work for the uuid.